qa/integration-tests/pom.xml

```diff
@@ -14,7 +14,24 @@
 
   <name>Camunda QA Integration Tests</name>
 
+  <properties>
```

Comment:
Versions should be in parent/pom.xml
Shouldn't all testcontainers dependencies have the same version?